src/ActiveField.php

```diff
@@ -216,6 +215,9 @@ public function checkbox($options = [], $enclosedByLabel = false)
             Html::removeCssClass($this->labelOptions, $this->horizontalCssClasses['label']);
             Html::addCssClass($this->wrapperOptions, $this->horizontalCssClasses['offset']);
         }
+        if (isset($options['template'])) {
```

Comment:
This condition is unnecessary - calling `unset()` on non-existing key does not trigger any error, so from performance perspective it is better call `unset($options['template']);` blindly without any `if`.

Reply:
I'll change